new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -676,16 +682,21 @@ public static boolean createPvc(V1PersistentVolumeClaim persistentVolumeClaim) t
       throw new IllegalArgumentException(
           "'namespace' field in the metadata cannot be null when calling createPvc()");
     }
-
+    logger.info("creating persistent volume claim" + persistentVolumeClaim.getMetadata().getName());
```

Comment:
Our logger uses string format type messages.  This is okay, but you could have written it as:
logger.info("creating persistent volume claim {0}", persistentVolumeClaim.getMetadata().getName())

Reply:
fixed it.